docs/examples/redis.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Using Redis with Wing"
+subtitle: "Example application that uses Redis to set cache values from a queue"
+type: 
+  - "pattern"
+platform:
+  - "awscdk"
+  - "sim"
+language:
+  - "wing"
+githubURL: "https://github.com/winglang/examples.git && cd examples/examples/redis"
+coverImage: "/img/examples/redis.png"
```

Comment:
Where are these images found?

Reply:
Good question, they are in docssite repo, but this does not make much sense. I will add them to this repo nice find.